shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -0,0 +1,572 @@
+
+\noindent
+Several parameters are pre-determined by external factors. These include the
+total \ada{} that could ever be created $A_T$;
+the total \ada{} that is in \emph{circulation} when the Shelley system launches
+(i.e. all \ada{} that is held by any entity on the launch date), $A_c$;
+and the \ada{} that is held in reserve when the system starts.
+These values are fixed and will not change.
+
+\clearpage
+\subsection{Stakepools}
+
+\subsubsection*{Operator-Set Parameters}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Range} & \textbf{Description} \\\hline
+$SP_c$ &  & Cost per day in \ada{} \\\hline
+$SP_m$ &  0\%-100\% & Percentage fee on rewards (``margin'') \\\hline
+\end{tabular}
+\end{center}
+\caption{Key Parameters that are set by the StakePool Operator.}
+\end{figure}
+
+\noindent
+The two main parameters that need to be set by the \emph{operator} are the cost per day that will be charged to
+the pool (in \ada), $SP_c$; and the percentage of any rewards that will subsequently be taken as a fee, $SP_m$.
+The cost is subtracted from the total rewards that are earned by the pool before any rewards are distributed.
+These parameters are advertised by the stakepool and used as part of the public ranking scheme.
+
+\subsubsection*{The \ada{} that is controlled by the Stakepool}
+
+\begin{figure}[h!]
+\begin{center}
+\begin{tabular}{||l|p{6cm}|l||}
+  \hline \hline
+\textbf{Parameter} & \textbf{Description} & \textbf{Calculated as} \\\hline
+$SP_p$ & \ada{} that is pledged to the Stakepool by the Owner(s) & \\\hline
+$SP_d$ & \ada{} that is delegated to the Stakepool & \\\hline
+$SP_T$ & All \ada{} that is controlled by the Stakepool & $SP_p + SP_d$ \\\hline
+$SP_F$ & Fraction of the \ada{} in circulation that is controlled by the Stakepool & {\large $\frac{SP_T}{A_c}$} \\\hline
+\end{tabular}
+\end{center}
+\caption{The \ada{} that is controlled by a StakePool.}
+\end{figure}
+
+\noindent
+The rewards that are received by the Stakepool will be proportional to the amount of \ada{} that it controls.
+This is calculated as the sum of the \ada{} that is \emph{pledged} to the Stakepool by the owners(s) plus
+the additional \ada{} that is \emph{delegated} to the Stakepool.  An owner may \emph{delegate} to a Stakepool if
+they wish as well as \emph{pledging} to it, but would receive lower rewards.  In return for receiving
+higher rewards, \emph{pledging} incurs higher risk, requiring owners to \emph{trust} each other.
+
+
+\section{The Simplified Incentives Scheme (Shelley Incentived Testnet)}
+\label{sec:testnet}
+
+The simplified scheme calculates rewards based on the total number of blocks that each stake pool produces,
+and distributes a fixed amount of \ada{} per epoch.
+Since \cardano{} is based on \emph{proof of stake}, then on average, each stake pool will produce
+rewards that are proportional to the stake that it holds.  So, if e.g. a pool holds 1\% of the total
```

Comment:
I don't think "produce" is the right word here. Maybe better "receive" ?

Reply:
OK